src/EditorFeatures/Core/Shared/Extensions/IWpfTextViewExtensions.cs

```diff
@@ -1,16 +1,20 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Windows.Media;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.CodeAnalysis.Editor.Shared.Extensions
 {
     internal static class IWpfTextViewExtensions
     {
-        public static void SizeToFit(this IWpfTextView view)
+        public static void SizeToContentsAndScale(this IWpfTextView view, double scaleFactor = 0.75)
```

Comment:
Nit: We don't normally make parameters optional in things that aren't part of the public API.